packages/core/src/browser/icons/CollapseAll.svg

```diff
@@ -1,7 +1,4 @@
-<!--Copyright (c) Microsoft Corporation. All rights reserved.-->
-<!--Copyright (C) 2018 TypeFox and others.-->
-<!--Licensed under the MIT License. See License.txt in the project root for license information.-->
-<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" enable-background="new -1 0 16 16" viewBox="-1 0 16 16">
-    <path fill="#424242" d="M14 1v9h-1v-8h-8v-1h9zm-11 2v1h8v8h1v-9h-9zm7 2v9h-9v-9h9zm-2 2h-5v5h5v-5z"/>
-    <rect width="3" height="1" x="4" y="9" fill="#00539C"/>
-</svg>
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Unless/until we put in place a [mechanism](https://github.com/eclipse-theia/theia/pull/6993#issuecomment-579782314) for us to consume these icons without copying them to our repo (e.g. publish the whole set on npm ourselves and then consume through the npm package), I think for each icon we need to add copyright headers, as were in place before. e.g.: 

```
<!--Copyright (c) Microsoft Corporation. All rights reserved.-->
<!--Copyright (C) 2020 TypeFox and others.-->
<!--Licensed under the MIT License. See License.txt in the project root for license information.-->
```

We're adding the Microsoft copyright just so it's clear who the copyright owner is, and the TypeFox license is because we changed the icons by adding the MS copyright :)

Reply:
So add that header to all the icons?

Reply:
yes, please.

Reply:
(assuming your work is owned by TypeFox - if not you'd adjust the second one to name whomever does)

Reply:
Got it!

Reply:
Done!